docs/_content/standaard/verzoeken/index.md

```diff
@@ -0,0 +1,46 @@
+# Inleiding
+Een verzoek is een aanvraag of opdracht aan de gemeente voor de levering van een product of dienst. Verzoeken vormen dus de schakel tussen de klant, producten en diensten van de gemeente zoals in de PDC staan, en het zaakgericht werken voor de afhandeling. Dit biedt een aantal voordelen, onder andere op het gebied van herkenbaarheid voor de klant, die niet meer wordt geconfronteerd met voor hem onherkenbare zaaktypen. Voor gemeenten ontstaat bovendien meer flexibiliteit: een aanvraag van een klant leidt niet langer vanzelfsprekend tot een zaak, en één of meerdere verzoeken kunnen worden gerelateerd aan één of meerdere zaken.
+
+Verzoeken zijn voor het eerst geintroduceerd in GFO-Zaken, maar is daarna weer verdwenen in het opvolgende informatiemodel RGBZ. In de visie van VNG Realisatie is dat echter een omissie, en is er behoefte aan een concept dat vooraf gaat aan de daadwerkelijke behandeling in een zaak (of andere activiteit).
+
+# Intake
+Op een verzoek wordt intake gedaan. Daarbij wordt het verzoek niet inhoudelijk beoordeeld, maar wordt wel gekeken of de  aangevraagde producten geleverd kunnen worden en of alle benodigde gegevens zijn ingevuld. Na de intake is het verzoek in behandeling genomen dan wel buiten behandeling gesteld. Een verzoek kan uitmonden in een of meerdere zaken of andere activiteiten (d.w.z. niet zaakgericht), dan wel direct worden afgehandeld, bijv. een product zoals een folder dat direct kan worden geleverd. Ook kan een verzoek worden gekoppeld aan reeds lopende zaak. Denk bijvoorbeeld aan een Melding Openbare Ruimte die reeds eerder is gedaan en waarvoor reeds een zaak loopt.
```

Comment:
> Een verzoek kan uitmonden in een of meerdere zaken of andere activiteiten (d.w.z. niet zaakgericht), dan wel direct worden afgehandeld, bijv. een product zoals een folder dat direct kan worden geleverd.

Als iets direct geleverd kan worden, wat voor status geven we dan aan het verzoek? Ook in relatie tot de tekst over statussen.

Reply:
Goede vraag. Ik denk dat het verzoek naar status "afgehandeld" gaat.

Reply:
Lijkt me goed als uitgelegd wordt waarom andersoortige verzoeken nooit de status 'afgehandeld' krijgen. Ik kan me nl. voorstellen dat die vraag gaat komen.